### DISCUSSION TITLE
Can't update to latest version of TRIQS

### DISCUSSION THREAD

**Discussion by AQuantumGuy** [TRIQS/triqs#849] [Q&A] [answered]
Hi, 
I'm trying to install Solid_DMFT, howerver it requires a TRIQS version of 3.1 at least. I downloaded TRIQS on my VM with command 

sudo apt-get update && sudo apt-get install -y triqs

However, the version I get is 3.0.2 and on the command it says that one is the latest one, however on the TRIQS website, it says that the latest is 3.1.1.   So my question is how can I update it to newer version so that I can work with both TRIQS and Solid_DMFT?

**Comment by the-hampel** [comment, marked as answer]
Hi, 
I remember that you are using a VM with Quantum Espresso. Are you using the Quantum Mobile VM? The problem is that this VM is running Ubuntu 18.04 which is by now fairly old and TRIQS does only support Ubuntu 18.04 up to 3.0.x . For newer version of triqs you have to update to a newer Ubuntu. On Ubuntu 20.04 or 22.04 you will find the 3.1.x packages. 

If you cannot update Ubuntu you can also use an older version of solid_dmft. As explained here https://flatironinstitute.github.io/solid_dmft/install.html you can checkout the `3.0.x` branch before installing. This should work with the triqs you have in the VM right now.

**Reply by AQuantumGuy** [reply]
Hi Alex,
Thank you for informing me that. I successfuly managed to download TRIQS newest version (3.1.x) after running it on Ubuntu 22.04 version. SInce Quantum Mobile VM was last updated last year, it contained many old packages. Now I'm only using VM of Ubuntu and downloaded all packages of TRIQS and Quantum Espresso. You're always helpful. 

Take care! :)
